docs/playbooks/ocp.md

```diff
@@ -1,87 +1,107 @@
 # OCP Playbooks
 
-## ROKS
-Provide intro ...
+## Provision
 
-### Required environment variables
+### ROKS
+As part of provisioning on IBM Cloud:
+
+- The **IBM operator catalog** will be installed
+- MAS development catalogs will be installed if `W3_USERNAME` and `ARTIFACTORY_APIKEY` environment variables are **both** defined
+- [Certificate manager](https://cert-manager.io) and [service binding](https://github.com/redhat-developer/service-binding-operator) operators will be installed at cluster scope
+- [Monitoring for user-defined projects](https://docs.openshift.com/container-platform/4.6/monitoring/enabling-monitoring-for-user-defined-projects.html#enabling-monitoring-for-user-defined-projects_enabling-monitoring-for-user-defined-projects) will be enabled in the cluster
+
+#### Required environment variables
 - `IBMCLOUD_APIKEY` Short statement
```

Comment:
Seeing as you're updating this doc page, can you fill in the missing content please ... would be nice if it's not just me fixing up all the doc :) 

Examples:

- `IBMCLOUD_APIKEY` Short statement
- export VAR_NAME=xxx
- As part of provisioning on QuickBurn: (sentence stops before it starts)